spec/cases/graph_api_spec.rb

```diff
@@ -94,6 +94,38 @@
         expect(@api).to receive(:api).with(path, {}, 'get', {})
         @api.graph_call(path)
       end
+
+      describe "the http_component option" do
+        it "sends the http_component option to the API call" do
```

Comment:
this test would've passed before changes right? basically just testing that the options get sent to api if passed in